test_scripts/LowVoltage/004_App_registered_30_sec_after_Wake_Up.lua

```diff
@@ -0,0 +1,89 @@
+---------------------------------------------------------------------------------------------------
+--
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/LowVoltage/common')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local hashId = { }
+local grammarId = { }
+
+--[[ Local Functions ]]
+local function addResumptionData()
+  local pAppId = 1
+  local cid = common.getMobileSession(pAppId):SendRPC("AddCommand", { cmdID = 1, vrCommands = { "OnlyVRCommand" }})
```

Comment:
@dboltovskyi Current function can be simplified in case refactoring proposed above.

Reply:
@aderiabin Accepted. Please check af2e393